craft_providers/lxd/launcher.py

```diff
@@ -72,48 +105,96 @@ def _create_instance(
     """
     logger.info("Creating new instance from remote")
     logger.debug(
-        "Creating new instance from image %r from remote %r.", image_name, image_remote
+        "Creating new instance from image %r from remote %r", image_name, image_remote
     )
-    instance.launch(image=image_name, image_remote=image_remote, ephemeral=ephemeral)
-    base_configuration.setup(executor=instance)
-
-    _set_timezone(instance, project, remote, lxc)
-
-    # return early if base instances and user id mapping are not specified
-    if not base_instance and not map_user_uid:
-        return
-
-    # the instance needs to be stopped before copying or updating the id map
-    instance.stop()
 
+    # Lockable base instance creation. Only one caller can create the base instance.
+    # Other callers will will get LXDError and wait until the base instance is created.
     if base_instance:
-        logger.info("Creating new base instance from instance")
+        logger.info("Preparing base instance from remote")
```

Comment:
pedantic: `Creating new` is used in the other log messages

```suggestion
        logger.info("Creating new base instance from remote")
```